### PR TITLE
BAH-1971 | Getting requester information on the reports table

### PR DESCRIPTION
**Pull request by abinaya-u**
## Requirements

- [x] This PR has a proper title that briefly describes the work done
- [x] I have squashed / amended the comments to make it more redable
- [x] I have included link to all the JIRA ticket('s)
- [x] I wrote the code as a pair or atleast performed a self-review of my own code
- [x] I have updated the documentation on [Bahmni Wiki](https://bahmni.atlassian.net/wiki/spaces/BAH/overview) or [README](https://github.com/Bahmni/bahmni-lab-frontend/blob/main/README.md) (if applicable)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Summary
Getting requester information on the reports table

## JIRA tickets
https://bahmni.atlassian.net/browse/BAH-1971?atlOrigin=eyJpIjoiOTNkMzQwNjBhMjMzNGZiZWEzNWY5MWY2YTEyMWRjZmQiLCJwIjoiaiJ9

### CODE REVIEW

File: src/__mocks__/reportTable.mock.ts [hunk 1]
```diff
@@ -60,6 +60,7 @@ export const mockReportTableResponse: ReportTableFetchResponse = {
               creation: '2022-05-20T20:00:00+11:00',
             },
           ],
+          performer: [{display: 'Super Man (Identifier: superman)'}],
```
Comment (N0-man) [inline src/__mocks__/reportTable.mock.ts:63]: when you add such mock data the current test would continue to work and no-one would know that the round brackets are an important consideration here that gets removed in the getRequestor function. It also took me a while to understand where the getRequestor is being testing and what are the requirements. Can you refactor this to articulate better in the test?

Comment (abinaya-u) [reply src/__mocks__/reportTable.mock.ts:63]: Added a test to check only provider name is displayed

File: src/report-table/report-table.component.tsx [hunk 2]
```diff
@@ -56,7 +56,16 @@ const ReportTable = (props: ReportTableProps) => {
 
   const getRequester = (
     performer: undefined | {display: string; reference: string},
-  ) => (performer && performer[0]?.display) || selfPatient
+  ) => {
+    if (performer) {
+      const lastIndex = performer[0]?.display?.lastIndexOf(' (')
```
Comment (sivareddyp) [inline src/report-table/report-table.component.tsx]: we can use regex pattern to replace.
`performer[0]?.display.replace(/\s\(.+\)/gm, "")`

Comment (abinaya-u) [reply src/report-table/report-table.component.tsx]: changes are made

**Pull request merged**